Tests/ApolloCodegenTests/CodeGeneration/Templates/TemplateRenderer_OperationFile_Tests.swift

```diff
@@ -359,6 +365,32 @@ class TemplateRenderer_OperationFile_Tests: XCTestCase {
     expect(actual).to(equalLineByLine(expected, atLine: 6, ignoringExtraLines: true))
   }
 
+  func test__moduleImports__givenValues_shouldGenerateImportStatements() {
+    // given
+    let config = buildConfig(
+      moduleType: 
+          .embeddedInTarget(
```

Comment:
It looks as though the custom imports should be rendered regardless of the `moduleType` configuration - it might be worth having additional tests for the other module types to ensure that remains true.